modules/cnf-image-based-upgrade.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-understanding-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept-stages_{context}"]
+= Stages of the image-based upgrade
+
+After generating the seed image on the seed cluster, you can move through the stages on the target cluster by setting the `spec.stage` field to the following values in the `ImageBasedUpgrade` CR:
+
+* `Idle`
+* `Prep`
+* `Upgrade`
+* `Rollback` (Optional)
+
+image::../images/696_OpenShift_Lifecycle_Agent_0624_0.png[Stages of the image-based upgrade]
+
+[id="ztp-image-based-upgrade-concept-idle_{context}"]
+== Idle stage
+
+The {lcao} creates an `ImageBasedUpgrade` CR set to `stage: Idle` when the Operator is first deployed.
+This is the default stage, there is no ongoing upgrade and the cluster is ready to move to the `Prep` stage.
+
+image::../images/696_OpenShift_Lifecycle_Agent_0624_1.png[Transition from Idle stage]
+
+You also move to the `Idle` stage to do one of the following steps:
+
+* Finalize a successful upgrade
+* Finalize a rollback
+* Cancel an ongoing upgrade until the pre-pivot phase in the `Upgrade` stage
+
+Moving to the `Idle` stage ensures that the {lcao} cleans up resources, so the cluster is ready for upgrades again.
+
+image::../images/696_OpenShift_Lifecycle_Agent_0624_2.png[Transitions to Idle stage]
+
+[IMPORTANT]
+====
+If using {rh-acm} when you cancel an upgrade, you must remove the `import.open-cluster-management.io/disable-auto-import` annotation from the target managed cluster to re-enable the automatic import of the cluster.
+====
+
+[id="ztp-image-based-upgrade-concept-prep_{context}"]
+== Prep stage
+
+[NOTE]
+====
+You can complete this stage before a scheduled maintenance window.
+====
+
+For the `Prep` stage, you specify the following upgrade details in the `ImageBasedUpgrade` CR:
+
+* seed image to use
+* resources to back up
+* extra manifests to apply and custom catalog sources to retain after the upgrade, if any
+
+Then, based on what you specify, the {lcao} prepares for the upgrade without impacting the current running version.
+During this stage, the {lcao} ensures that the target cluster is ready to proceed to the `Upgrade` stage by checking if it meets certain conditions and pulls the seed image to the target cluster with additional container images specified in the seed image.
+
+You also prepare backup resources with the OADP Operator's `Backup` and `Restore` CRs.
+These CRs are used in the `Upgrade` stage to reconfigure the cluster, register the cluster with {rh-rhacm}, and restore application artifacts.
+
+Additionally to the OADP Operator, the {lcao} uses the `ostree` versioning system to create a backup, which allows complete cluster reconfiguration after both upgrade and rollback.
```

Comment:
This needs to be reworded. ostree is not creating any backups. 
The new version is installed on the disk during the prep stage as an ostree stateroot.
During the upgrade LCA will backup low level platform configuration outside of OADP which is restored after booting into the new stateroot.